.gitignore

```diff
@@ -7,6 +7,8 @@
 /.release
 /.tarballs
 
+.vimlocal
```

Comment:
I removed editor specific ignores various repos some time ago. There's just too many of them and they get more over time. These should be in every users global `.gitignore`.